src/groups/mqb/mqbstat/mqbstat_queuestats.h

```diff
@@ -419,6 +420,104 @@ struct QueueStatsUtil {
                                  bmqst::StatContext*            statContext);
 };
 
+// -----------------------
+// struct DomainQueueStats
+// -----------------------
+
+/// Namespace for the constants of stat values that applies to the queues
+/// on the domain
+struct DomainQueueStats {
```

Comment:
Also might be a private type for `QueueStatsDomain`

Reply:
Can not make it private, because it's used in QueueStatsUtil::initializeTableAndTipDomains
My suggestion is to put `struct DomainQueueStats` into public section in `class QueueStatsDomain`

Reply:
Can do it in the next PR